### PR TITLE
Delete wrong cleaning info

### PR DESCRIPTION
**Pull request by sean-kang**
The CRC checksums added by me need to be all removed. I originally thought "Clean" meant the file was clean in terms of the integrity of the file.  It seems that "Clean" had its own meaning in Skyrim modding context. Sorry about that.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -2497,9 +2497,6 @@ plugins:
       # version: 3.0.0
       - crc: 0x96BF25A6
         util: 'SSEEdit v4.0.0'
-      # version: 3.2.0
-      - crc: 0x747BD772
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:2502]:
```yaml
  - name: 'Audio Overhaul Skyrim.esp'
    dirty:
      # version: 3.2.0
      - <<: *quickClean
        crc: 0x747BD772
        util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 6
    clean:
      # version: 3.2.0, after QAC
      - crc: 0x0CD5C85F
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -3455,10 +3452,6 @@ plugins:
           - 'Interesting NPCs'
           - '[ RS Children Patch Compendium ](https://www.nexusmods.com/skyrimspecialedition/mods/13409)'
         condition: 'active("3DNPC.esp") and not active("RSC 3DNPC Patch.esp")'
-    clean:
-      # version: 1.1.2
-      - crc: 0xE8137869
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:3461]: That one is fine.
```yaml
  - name: 'RSkyrimChildren.esm'
    clean:
      # version: 1.1.2
      - crc: 0xE8137869
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -5062,33 +5055,12 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Campfire - Complete Camping System' ]
         condition: 'active("Campfire.esm") and not active("Archery_Campfire_Patch.esp")'
-    clean:
-      # version: 2.5
-      - crc: 0xB2AD660F
-        util: 'SSEEdit v4.0.2'
-      # version: 2.4
-      - crc: 0x0DDC51B6
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5071]:
```yaml
  - name: 'ArcheryV2.esp'
    dirty:
      # version: 2.4
      - <<: *quickClean
        crc: 0x0DDC51B6
        util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 4
      # version: 2.5
      - <<: *quickClean
        crc: 0xB2AD660F
        util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 4
```

ESL Version is incorrectly flagged as an `ESL`.

```
[00:00] Saving: ArcheryV2.esp.save.2019_06_15_02_38_13
[00:00] Error saving ArcheryV2.esp.save.2019_06_15_02_38_13: Record 1stPersonWarBow [STAT:FE00037C] has invalid ObjectID 02F37C for a light module. You will not be able to save this file with ESL flag active
[00:00] Errors have occured. At least one file was not saved.
```

File: masterlist.yaml [hunk 4]
```diff
@@ -5062,33 +5055,12 @@ plugins:
...
   - name: 'Archery_Bruma_Patch.esp'
     req:
       - 'ArcheryV2.esp'
-    clean:
-      # version: 1.2
-      - crc: 0x0470CE8E
-        util: 'SSEEdit v4.0.2'
-      # version: 1.1 Aleyid Arrows
-      - crc: 0xEF057475
-        util: 'SSEEdit v4.0.2'
```
Comment (CptMcSplody) [inline masterlist.yaml]: Noth of those are fine.
```
  - name: 'Archery_Bruma_Patch.esp'
    clean:
      - crc: 0xEF057475
        util: 'SSEEdit v4.0.2'
      - crc: 0x0470CE8E
        util: 'SSEEdit v4.0.2'
```

Comment (MacSplody) [inline masterlist.yaml:5081]: Both of those are fine.
```
  - name: 'Archery_Bruma_Patch.esp'
    clean:
      - crc: 0xEF057475
        util: 'SSEEdit v4.0.2'
      - crc: 0x0470CE8E
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 5]
```diff
@@ -5062,33 +5055,12 @@ plugins:
...
   - name: 'Archery_Campfire_Patch.esp'
     req:
       - 'ArcheryV2.esp'
-    clean:
-      # version: 1.2
-      - crc: 0xBE199860
-        util: 'SSEEdit v4.0.2'
-      # version: 1.1 Realisitc Arrows
-      - crc: 0x556C229B
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5091]: Also correct.
```yaml
  - name: 'Archery_Campfire_Patch.esp'
    clean:
      - crc: 0x556C229B
        util: 'SSEEdit v4.0.2'
      - crc: 0xBE199860
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 6]
```diff
@@ -5450,10 +5422,6 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'Weapons Armor Clothing and Clutter Fixes' ]
         condition: 'active("WACCF_Armor and Clothing Extension.esp") and not active("EconomyOverhaul LV - WACCF Patch.esp")'
-    clean:
-      # version: 2.1
-      - crc: 0xBBA1C2CA
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5456]: Also correct.
```yaml
  - name: 'EconomyOverhaulandSpeechcraftImprovementsLite.esp'
    clean:
      - crc: 0xBBA1C2CA
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 7]
```diff
@@ -5768,10 +5736,6 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'Wildcat' ]
         condition: 'active("Wildcat - Combat of Skyrim.esp") and not active("Immersive Movement - Wildcat Patch.esp")'
-    clean:
-      # version: 1.3.2
-      - crc: 0xB91C4616
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5774]: Also fine.
```yaml
  - name: 'Immersive Movement.esp'
    clean:
      - crc: 0xB91C4616
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 8]
```diff
@@ -5781,26 +5745,14 @@ plugins:
...
-      - crc: 0xDCF50B37
-        util: 'SSEEdit v4.0.2'
   - name: 'Immersive Movement - Smilodon Patch.esp'
     req:
       - 'Immersive Movement.esp'
       - 'Smilodon - Combat of Skyrim.esp'
-    clean:
-      # version: 1.3.2
-      - crc: 0x9906E4A8
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5795]:
```yaml
  - name: 'Immersive Movement - Smilodon Patch.esp'
    clean:
      - crc: 0x9906E4A8
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 9]
```diff
@@ -5781,26 +5745,14 @@ plugins:
...
   - name: 'Immersive Movement - Wildcat Patch.esp'
     req:
       - 'Immersive Movement.esp'
       - 'Wildcat - Combat of Skyrim.esp'
-    clean:
-      # version: 1.3.2
-      - crc: 0x82AA7490
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5803]:
```yaml
  - name: 'Immersive Movement - Wildcat Patch.esp'
    clean:
      - crc: 0x82AA7490
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 10]
```diff
@@ -5781,26 +5745,14 @@ plugins:
       - 'Mortal Enemies - Rival Remix.esp'
       - 'Mortal Enemies - Rival Remix - NoRunWalkChanges.esp'
       - 'Mortal Enemies - 2.0 Test Version.esp'
-    clean:
-      # version: 1.3.2
-      - crc: 0xDCF50B37
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:5787]:
```yaml
  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
    clean:
      - crc: 0xDCF50B37
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 11]
```diff
@@ -6958,10 +6910,6 @@ plugins:
     msg:
       - <<: *compatNotes
         subs: [ 'https://www.nexusmods.com/skyrimspecialedition/articles/119' ]
-    clean:
-      # version: 2.00SSE
-      - crc: 0x70092444
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:6964]:
```yaml
  - name: 'Smilodon - Combat of Skyrim.esp'
    clean:
      - crc: 0x70092444
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 12]
```diff
@@ -7494,25 +7442,12 @@ plugins:
...
-      # version: 1
-      - crc: 0x135E09DE
-        util: 'SSEEdit v4.0.2'
   - name: 'imp_helm_USSEP.esp'
     req:
       - 'Unofficial Skyrim Special Edition Patch.esp'
-    clean:
-      # version: 1
-      - crc: 0x56ED0191
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:7515]:
```yaml
  - name: 'imp_helm_USSEP.esp'
    clean:
      - crc: 0x56ED0191
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 13]
```diff
@@ -7494,25 +7442,12 @@ plugins:
...
-      # version: 1
-      - crc: 0x5C4A5C28
-        util: 'SSEEdit v4.0.2'
   - name: 'imp_helm_IDPM.esp'
     after:
       - 'imp_helm_USSEP.esp'
-    clean:
-      # version: 1
-      - crc: 0x135E09DE
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:7508]:
```yaml
  - name: 'imp_helm_IDPM.esp'
    clean:
      - crc: 0x135E09DE
        util: 'SSEEdit v4.0.2'
```

File: masterlist.yaml [hunk 14]
```diff
@@ -7494,25 +7442,12 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
         name: 'Improved Closedfaced Helmets Patches'
-  - name: 'imp_helm_Immersive_Armors.esp'
-    clean:
-      # version: 1
-      - crc: 0x5C4A5C28
-        util: 'SSEEdit v4.0.2'
```
Comment (MacSplody) [inline masterlist.yaml:7501]:
```yaml
  - name: 'imp_helm_Immersive_Armors.esp'
    clean:
      - crc: 0x5C4A5C28
        util: 'SSEEdit v4.0.2'
```

**Comment by MacSplody** [comment]
Most of them are correct, just Audio Overhaul and Archery that need correction.
I've updated these in a separate commit.

File: masterlist.yaml (hunk 3 above)
Comment (MacSplody) [reply masterlist.yaml:5071]: That invalid `ESL` flag will be picked up by LOOT.

image

**Pull request closed without merging**